gspread/worksheet.py

```diff
@@ -402,6 +403,15 @@ def get_all_records(
 
         keys = data[idx]
 
+        # Check keys are uniques
+        # do not use 'headers' in return values
+        # it is not ordered
+        headers = set()
+        for key in keys:
+            if key in headers:
+                raise GSpreadException("headers must be uniques")
+            headers.add(key)
```

Comment:
If headers aren't used later I think this part could be simplified:
```
if len(keys) != len(set(keys)):
    raise GSpreadException("headers must be uniques")
```

Reply:
thank you very much for you review, I really appreciate.
I update the code to use your proposal.